src/com/google/common/css/compiler/ast/GssParserCC.jj

```diff
@@ -707,6 +707,8 @@ PARSER_END(GssParserCC)
   // Special handling needed because ':not' takes simple selectors as
```

Comment:
`not` supports compounding only through duplication:

`div:not(.class1):not(.class2)`